testcases/OpTestInbandIPMI.py

```diff
@@ -73,6 +73,18 @@ def run_ipmi_cmd_on_host(self, i_cmd):
             raise OpTestError(l_msg)
         return l_res
 
+class BasicInbandIPMI(OpTestInbandIPMIBase):
+    ##
+    # @brief  It will check basic channel functionalities: info and authentication capabilities.
+    #
+    # @return l_res @type list: output of command or raise OpTestError
+    #
+    def test_channel(self):
+        print "Inband IPMI[OPEN]: Channel tests"
+        self.run_ipmi_cmd_on_host(BMC_CONST.IPMITOOL_OPEN + BMC_CONST.IPMI_CHANNEL_AUTHCAP)
+        self.run_ipmi_cmd_on_host(BMC_CONST.IPMITOOL_OPEN + BMC_CONST.IPMI_CHANNEL_INFO)
```

Comment:
Huh... that's unusual. It works flawlessly for me. Anyway, I've moved it around as suggested in 0bced281b3a3